modules/dnn/test/test_onnx_importer.cpp

```diff
@@ -516,9 +516,34 @@ TEST_P(Test_ONNX_layers, Broadcast)
     testONNXModels("channel_broadcast", npy, 0, 0, false, true, 2);
 }
 
-TEST_P(Test_ONNX_layers, DynamicResize)
+TEST_P(Test_ONNX_layers, DynamicResizeOpset9)
 {
-    testONNXModels("dynamic_resize", npy, 0, 0, false, true, 2);
+    testONNXModels("dynamic_resize_9", npy, 0, 0, false, true, 2);
+}
+
+TEST_P(Test_ONNX_layers, DynamicResizeOpset10)
+{
+    testONNXModels("dynamic_resize_10", npy, 0, 0, false, true, 2);
+}
+
+TEST_P(Test_ONNX_layers, DynamicResizeOpset11)
+{
+    testONNXModels("dynamic_resize_11", npy, 0, 0, false, true, 2);
+}
+
+TEST_P(Test_ONNX_layers, DynamicResizeScaleOpset9)
+{
+    testONNXModels("dynamic_resize_scale_9", npy, 0, 0, false, true, 2);
+}
+
+TEST_P(Test_ONNX_layers, DynamicResizeScaleOpset10)
+{
+    testONNXModels("dynamic_resize_scale_10", npy, 0, 0, false, true, 2);
+}
+
+TEST_P(Test_ONNX_layers, DynamicResizeScaleOpset11)
+{
+    testONNXModels("dynamic_resize_scale_11", npy, 0, 0, false, true, 2);
 }
```

Comment:
I think it's better to combine all the tests into one

Reply:
Usually it is very hard to investigate and debug combined tests.

Each test should be simple enough.

Reply:
mmhh I just grouped them before seeing your comment @alalek